docs/boshdirector.adoc

```diff
@@ -1,20 +1,32 @@
-= Installing Bosh Director
+= Configuring Ops Manager Director
 Eitan Suez <esuez@pivotal.io>
 v0.1, 2017
 :project_id: {{project_id}}
 :domain_name: {{domain_name}}
 :env_name: {{env_name}}
 :director_ip_address: {{director_ip_address}}
+:region: {{region}}
 
+Estimated time:  30 minutes
+
+= Requirements
+
+Lab Requirements
+
+= What you will learn
+
+* How to configure and install Ops Manager Director
+
+= Exercises
 
 The terraform scripts we ran in the last lab created the Ops Manager VM.
 
 The two main tasks we will accomplish in this lab are:
 
-. setup access to Ops Manager, by selecting a username and password (and passphrase)
-. use the Ops Manager to configure the BOSH Director.
+. Setup access to Ops Manager, by selecting a username and password (and passphrase)
+. Use the Ops Manager to configure the Ops Manager Director (also referred to as the BOSH Director).
```

Comment:
ok, i think we agree here to stick to the correct name for this which is the ops mgr director.  will merge this change.